src/app/ixy-fwd.c

```diff
@@ -32,8 +33,8 @@ int main(int argc, char* argv[]) {
 		return 1;
 	}
 
-	struct ixy_device* dev1 = ixy_init(argv[1], 1, 1);
-	struct ixy_device* dev2 = ixy_init(argv[2], 1, 1);
+	struct ixy_device* dev1 = ixy_init(argv[1], 1, 1, 0x028, -1, true);
+	struct ixy_device* dev2 = ixy_init(argv[2], 1, 1, 0, 0, false);
```

Comment:
setting the ITR should be a separate function because it's specific to intel NICs

Reply:
Ok what I have done now is to remove the parameter and move it into the `ixgbe` driver.